src/utils/changes.ts

```diff
@@ -23,6 +23,7 @@ export interface Changeset {
 export interface ChangesetLoc {
   start: RegExpExecArray;
   end: RegExpExecArray | null;
+  padding: string;
```

Comment:
If I'm reading this all right, this is just a refactor, not a behavior change with regard to padding, correct?

Reply:
Correct 🙂